rust/rope/src/rope.rs

```diff
@@ -1240,4 +1256,19 @@ mod tests {
         assert!(long_text.len() > 1024);
         assert_eq!(cow, Cow::Borrowed(&long_text[..500]));
     }
+
+    #[test]
+    fn serialize_and_deserialize() {
+        use serde_json::{from_str, to_string};
+        const TEST_LINE: &str = "test line\n";
+
+        // repeat test line enough times to exceed maximum leaf size
+        let n_seg = MAX_LEAF / TEST_LINE.len() + 1;
+        let test_str = TEST_LINE.repeat(n_seg);
+
+        let rope = Rope::from(test_str.as_str());
+        let json = to_string(&rope).expect("error serializing");
```

Comment:
I was wondering where `to_string` comes from and had to search to find the explanation. It would be clearer to call it `serde_json::to_string`, and as a bonus the `use` statement can be removed.
```suggestion
        let json = serde_json::to_string(&rope).expect("error serializing");
```